sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -27,10 +27,14 @@
 For internal use only; no backwards-compatibility guarantees.
 """
 from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
 
-from types import NoneType
-
-import six
+from builtins import chr
+from builtins import int
```

Comment:
<!--new_thread; commit:05e95b6995f0b88a53930c30459c8d81e6692219; resolved:0-->
For my edification, is this the recommended way of using these types / methods in python 3?  Are they no longer in the global namespace?  It seems a bit verbose to have to import `object` like this in every file.

Reply:
This is for compatibility. On python 2, they import from future.builtins, on python 3 it has no effect.